docs/dev/README.md

```diff
@@ -1,35 +1,73 @@
+
+The [Integrations page][3] includes integrations built by both Datadog and our Technology Partners, available at no cost to Datadog customers. 
+
+The [Datadog Marketplace][4], on the other hand, is a commercial platform for Technology Partners to _sell_ a variety of offerings, including integrations, software licenses, and professional services to Datadog customers.
+
+## Datadog Integrations
+
+### [Agent-based Integration][2]
+Agent-based integrations use the Datadog Agent to submit data via checks written by the partner. The implementation code for these integrations is hosted by Datadog. Agent Integrations are best suited for collecting data from systems or applications that live in a local area network (LAN) or virtual private cloud (VPC). Writing an Agent integration requires you to publish and deploy your solution as a Python wheel (.whl).
+
+### [API-based Integration][1]
+API-based integrations can enrich a customer's Datadog environment by submitting data from external technologies via the Datadog API. Customers authorize API integrations access to their accounts through OAuth, and partners write and host the implementation code that makes up the integration. API integrations work well for partners building a connector between Datadog and another SaaS platform.
+
 ## Why create an integration?
 
-[Custom Checks][1] are great for occasional reporting, or in cases where the data source is either unique or very limited. For more general use-cases - such as application frameworks, open source projects, or commonly-used software - it makes sense to write an integration.
+**Correlate your data with user observability data** - Leverage Datadog to increase the value of your platform by  allowing customers to see the data from your platform alongside the rest of their technology stack.
+
+**Increase MTTR for customers** - When a customer's account is enriched with data from an integration, they are able to see a broader view of their entire stack, allowing them to debug and remediate issues more quickly. 
+
+**Increase Adoption and Visibility** - Ensuring native support for Datadog reduces friction to adoption, and displaying a tile on our Integrations page provides key visibility to all of Datadog's customers.
+
+**Submit out-of-the-box metrics at no extra cost** - Metrics reported from official Datadog integrations (unless the integration is sending in potentially unlimited metrics) are not counted as custom metrics, and therefore won't impact a customer's custom metric allocation.
+
+### Responsibilities
+Going forward, you, as the author of the integration, are the active maintainer of the integration. You are responsible for maintaining the code and ensuring the integration’s functionality. Datadog extends support on a best-effort basis for partner-built integrations, so please reach out to Datadog support if help is needed.
+
+## Join the Datadog partner network
```

Comment:
Can we reorder to:

Overview > Join the Datadog Partner Network > Why create an integration > Datadog integrations > Let's get started?

Could we also keep headings consistent with capitalizations? For example, "Datadog Partner Network" should be capitalized but only the "Let's" in "Let's Get Started" should be capitalized.